### DISCUSSION TITLE
Colorizer Issue Came Out of Nowhere [Noob please help]

### DISCUSSION THREAD

**Discussion by nsirimo** [NvChad/NvChad#1677] [Q&A]
![Screen Shot 2022-11-10 at 3 42 54 PM](https://user-images.githubusercontent.com/24559644/201229028-372db38c-9c31-4721-a1f4-5815d8e18472.png)

I don't know what I changed, I don't remember changing a thing so I'm not sure what would cause this to stop working. I'm not very familiar with anything beyond adding plugins and stuff so I'm not sure how to fix this. I've tried google but haven't found much to help.

**Comment by siduck** [comment]
@nsirimo please update neovim to 0.8 and update nvchad , do packersync. reopen nvim

**Reply by nsirimo** [reply]
So I followed your instructions and it got worse :(. Now I no longer have the dash when I first open nv (I think from alpha?) and the colorizer still doesn't work.

**Reply by siduck** [reply]
@nsirimo please  do know that alpha plugin is disabled by default, all you have to do is override its plugin definition and  please make issue for colorizer on its repo
